_posts/2018-02-26-success-with-serverless.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+status: publish
+published: true
+title: 'Success With Serverless'
+author:
+  display_name: Scott Davis
+  email: stdavis@utah.gov
+date: 2018-02-26 23:45:28
+categories:
+- Developer
+tags:
+- nodejs
+---
+
+In this post, I'd like to describe a recent success story that we've had using [serverless computing](https://en.wikipedia.org/wiki/Serverless_computing). This new(ish) technology has some exciting advantages over traditional solutions. We were looking for an excuse to give it a try and found a great fit.
```

Comment:
_"We were looking for an excuse to give it a try and found a great fit."_:  Could we reword to "We were looking for an excuse to give serverless computing a try, and we found it to be a great fit." For some reason, the "found a great fit" isn't quite clear to me. I think the "found it to be a great fit" with the specific reference to serverless computing makes it a little clearer though. Or maybe something like "We've been looking for an excuse to give serverless computing a try, and we weren't disappointed."

Reply:
I was trying (unsuccessfully, it seems) to say that the print-proxy service was a great fit for serverless computing. Is something like this more clear?
>We were looking for an excuse to give serverless computing a try, and found that this project was a great fit.

Reply:
Oh! Clearly I misunderstood! But I think (maybe) I'm understanding it more now. Just brainstorming here; what about something like: 

"In this post, I’d like to describe a recent success story that we’ve had using serverless computing. AGRC has been having some issues with a print-proxy service, and we realized that the situation provided us the perfect opportunity to try out serverless computing. This new(ish) technology has some exciting advantages over traditional solutions, and we've been looking for an excuse try it out. We weren't disappointed."